notebooks/Binary Classification on COMPAS dataset.ipynb

```diff
@@ -19,9 +19,30 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```

Comment:
Before committing notebooks we always clear the output. That should reduce the number of changes to a minimum here :-)